Src/Coravel/Scheduling/Schedule/Scheduler.cs

```diff
@@ -73,6 +73,13 @@ public IScheduleInterval ScheduleAsync(Func<Task> asyncTaskToSchedule)
             return scheduled;
         }
 
+        public IScheduleInterval ScheduleWithParams(Type invocableType, params object[ ] parameters)
```

Comment:
Minor code style: Could you remove the space within the `[ ]`? Thanks!

Reply:
Resolved  😄. It would be better if you could include `.editorconfig` to avoid such issues in the future from anyone.  It will assure you that it will remain same irrespective of the individual's styling preferences.